mgmtsystem_action/data/automated_reminder.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="ir_cron_mgmtsystem_action_reminder" model="ir.cron">
+            <field name="name">Email Action Reminders 10 days before due date.</field>
+            <field name="user_id" ref="base.user_root"/>
+            <field name="interval_number">1</field>
+            <field name="interval_type">days</field>
+            <field name="numbercall">-1</field>
+            <field eval="False" name="doall"/>
+            <field eval="'mgmtsystem.action'" name="model"/>
+            <field eval="'process_reminder_queue'" name="function"/>
```

Comment:
Please, i don't really understand what to do here. You can check if what i've done is correct